docs/README.md

```diff
@@ -25,23 +25,23 @@ Install these dependencies:
 
 ### MacOS
 
-1. Install [doxygen](http://www.stack.nl/~dimitri/doxygen/download.html).
-2. Make doxygen available on the command line.
-  1. Locate your doxygen executable.  One possible location is
-     `/Applications/Doxygen.app/Contents/Resources/doxygen`
-  2. Make that executable available in your path.  For example you could add a
-     symbolic link from `/usr/local/bin` to the doxygen executable:
-     `$ ln -s /Applications/Doxygen.app/Contents/Resources/doxygen /usr/local/bin/doxygen`
-3. Install [Boost](http://www.boost.org/users/download/)
+1. Install doxygen:
+  * Use homebrew to install: `brew install doxygen`.  The executable will be
+    installed in `/usr/local/bin` which is already in your path.
+  * Alternatively, install from here: [doxygen](http://www.stack.nl/~dimitri/doxygen/download.html).
+    You'll then need to make doxygen available to your command line.  You can
+    do this by adding a symbolic link from `/usr/local/bin` to the doxygen
+    executable.  For example, `$ ln -s /Applications/Doxygen.app/Contents/Resources/doxygen /usr/local/bin/doxygen`
+2. Install [Boost](http://www.boost.org/users/download/)
```

Comment:
An alternate option with is using homebrew again.
* Use homebrew to install: `brew install boost`.